Formula/boost-python.rb

```diff
@@ -12,9 +12,6 @@ class BoostPython < Formula
     sha256 "0d49ef79a0be83226cdb0fb2b010f6bcc867ce343fa980aee2ddb6356327688f" => :el_capitan
   end
 
-  option "without-python", "Build without python 2 support"
-
-  depends_on "python3" => :optional
```

Comment:
Wondering if this is worth a `revision` bump despite being not a default option. I presume if you're previously installed `boost` with `python3` and then go install `boost-python3` after this you're going to create link conflicts for yourself?

Reply:
Done